src/Controls/src/Core/Handlers/Items/ItemsViewHandler.cs

```diff
@@ -28,7 +28,10 @@ public ItemsViewHandler(PropertyMapper mapper = null) : base(mapper ?? ItemsView
 			[Controls.ItemsView.EmptyViewTemplateProperty.PropertyName] = MapEmptyViewTemplate,
 			[Controls.ItemsView.FlowDirectionProperty.PropertyName] = MapFlowDirection,
 			[Controls.ItemsView.IsVisibleProperty.PropertyName] = MapIsVisible,
-			[Controls.ItemsView.ItemsUpdatingScrollModeProperty.PropertyName] = MapItemsUpdatingScrollMode
+			[Controls.ItemsView.ItemsUpdatingScrollModeProperty.PropertyName] = MapItemsUpdatingScrollMode,
+			#if IOS || ANDROID
```

Comment:
The behavior is the expected on Catalyst and Windows?

Reply:
@jsuarezruiz ,
Yes, this behavior is expected on Windows. With the current iOS changes, it is now applicable to Mac as well